src/page-response-impact/ValidDataWrapper.tsx

```diff
@@ -0,0 +1,90 @@
+import { sum } from "d3";
+import { pick } from "lodash";
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+
+import Colors from "../design-system/Colors";
+import { Facilities, Facility } from "../page-multi-facility/types";
+const POPULATION_DATA_ERROR_MSG =
+  "Impact could not be generated because one or more of your facilities does not have a staff or incarcerated population count. Please add population numbers and try again.";
```

Comment:
Is it confirmed that we don't want to display the impact report if there are no staff numbers? It feels like we should still be able to generate a report given that the incarcerated population is significantly larger than the staff population, but if there are metrics that necessarily require both then that might make sense.

Reply:
Alternatively, if this is partially meant to be a forcing function to encourage users to provide staff info, I get that, too!

Reply:
The requirement to validate staff numbers was based on my recommendation, I thought that the model outputs were zero if there was no staff population input. As I'm playing around with it more this morning it looks like that is not the case, the zero outputs I was seeing must have been due to the prison population input. I removed the staff population check for now, @sychang if you want to keep it in we can just revert that commit.

Reply:
Yes that's not a requirement for the model, and poking around Firebase it looks like there are definitely some models where that input is not filled in. I'd suggest removing this particular validation.